source/reference/upgrade.txt

```diff
@@ -444,12 +453,22 @@ end-of-life (EOL).
 To learn more about the MongoDB support for EOL products,
 see the `Legacy Support Policy <https://www.mongodb.com/support-policy/legacy>`__.
 
+.. _java-server-release-change-v5.5:
+
+Driver Version 5.5 Server Support Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The v5.5 driver drops support for {+mdb-server+} v4.0.
+To use the v5.5 driver, you must connect to {+mdb-server+} v4.0 or later. To
```

Comment:
4.0 -> 4.2. also removed the v's

```suggestion
The v5.5 driver drops support for {+mdb-server+} 4.0.
To use the v5.5 driver, you must connect to {+mdb-server+} 4.2 or later. To
```

Reply:
good catch thanks! Updated the other similar text on the page as well.